### PR TITLE
Added architecture diagram

### PR DESCRIPTION
**Pull request by dendle**
### Context

Added high level overview of Find a lost TRN service, along with a technical diagram.

### Changes proposed in this pull request

A new markdown file and image in the docs folder

### Checklist

- [ ] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: docs/images/FindALostTRNDiagram.png:Zone.Identifier [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
+LastWriterPackageFamilyName=Microsoft.ScreenSketch_8wekyb3d8bbwe
+ZoneId=3
```
Comment (tvararu) [inline docs/images/FindALostTRNDiagram.png:Zone.Identifier:3]: What's this file for? Do we need it in version control?

**Review by tvararu — APPROVED** [review]
Needs `bin/lint` as well to run the linter :+1:

File: docs/technical-architecture.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Find a lost TRN - Technical Architecture
+
+## Overview
+
+The Find a lost TRN service is a public facing web application that allows its users to retrieve their TRN by supplying personal information.  It works by querying the DQT API with the supplied details, and returning them to the user.  As a fallback, if the DQT API is unavailable, the Find a lost TRN service will create a ticket in a ZenDesk system that the supporting team will then pick up, and handle manually.
```
Comment (benilovj) [inline docs/technical-architecture.md:5]:
```suggestion
The Find a lost TRN service is a public facing web application that allows its users to retrieve their TRN by supplying personal information.  It works by querying the DQT API with the supplied details, and returning them to the user.  As a fallback, if the DQT API is unavailable, the Find a lost TRN service will create a ticket in a Zendesk system that the supporting team will then pick up, and handle manually.
```

**Review by benilovj — APPROVED** [review]
LGTM.

Nitpick: do we want the Zone identifier file in there?

File: docs/images/FindALostTRNDiagram.png:Zone.Identifier (hunk 1 above)
Comment (dendle) [reply docs/images/FindALostTRNDiagram.png:Zone.Identifier:3]: Sorry mate, I have no idea
let me remove it - thanks for letting me know

**Pull request merged**